app/src/main/java/cm/aptoide/pt/permission/DialogPermissions.java

```diff
@@ -58,6 +60,18 @@ public static DialogPermissions newInstance(GetApp getApp) {
     return dialog;
   }
 
+  public static DialogPermissions newInstance(DetailedApp app) {
```

Comment:
Only parameters needed should be passed, the other method that as a GetApp should also be refactored so we can use a single static factory method.